<ui version="4.0" >
 <class>ContactProperties</class>
 <widget class="QDialog" name="ContactProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Nickname</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="propertiesTabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QFormLayout" name="formLayout" >
         <property name="labelAlignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="nicknameTitle" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Nickname</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLabel" name="nicknameLabel" >
           <property name="text" >
            <string>nicknameLabel</string>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="statuslineTitle" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Statusline</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLabel" name="statuslineLabel" >
           <property name="text" >
            <string>statuslineLabel</string>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="idTitle" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Service/User ID</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLabel" name="idLabel" >
           <property name="text" >
            <string>idLabel</string>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="close" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>ContactProperties</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel" >
     <x>95</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
